packages/components/affix/src/Affix.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+  <div ref="wrapperRef" :style="wrapperStyle" class="ix-affix-wrapper">
+    <div ref="affixRef" class="ix-affix" :style="affixStyle">
+      <slot></slot>
+    </div>
+  </div>
```

Comment:
```suggestion
  <div ref="affixRef" :style="affixStyle" class="ix-affix">
    <div ref="contentRef" class="ix-affix-content" :style="contentStyle">
      <slot></slot>
    </div>
  </div>
```

最外层的 class 名和组件同名，整体组件都保持这个风格吧。